### DISCUSSION TITLE
Missing state for US airports

### DISCUSSION THREAD

**Discussion by dlubom** [osm-search/Nominatim#3316] [Q&A] [answered]
Is it bug in Nominatim or some missing relation: https://community.openstreetmap.org/t/connecticut-airports-missing-state-relation/108434

I assume that all places in US should have state.

**Comment by mtmail** [comment, marked as answer]
The borders of Connecticut were broken. Already fixed my a mapper a couple of days ago (https://www.openstreetmap.org/changeset/146496661).

Nominatim by default has a limit how many places it will update per change (https://nominatim.org/release-docs/latest/customize/Settings/#nominatim_limit_reindexing).

Nominatim admins can kick of a full reindex of all addresses for [Connecticut](https://www.openstreetmap.org/relation/165794) with
```
nominatim refresh --data-area r165794
nominatim index
```

Afaik the admin for nominatim.osm.org is currently on holiday

cc @lonvia
